qds_sdk/commands.py

```diff
@@ -87,10 +87,24 @@ def run(cls, **kwargs):
         Returns:
             Command object
         """
+        err_pointer, tmp_pointer, new_bytes = 0, 0, 0
         cmd = cls.create(**kwargs)
         while not Command.is_done(cmd.status):
             time.sleep(Qubole.poll_interval)
             cmd = cls.find(cmd.id)
+            if kwargs.get('print_logs_live', False):
+                log, err_length, tmp_length = cmd.get_log_partial(err_pointer, tmp_pointer)
```

Comment:
Can you add this comment in the code. and edit if required. 

"""
log: Text string returned by the API.
err_length: The length of err file text (bytes).
tmp_length: The length of tmp file text which is present in the "log" var.
err_pointer: Stores the err text we have processed/recv until now and it gets passed to API in next file to indicate where to start from to get the new logs.
tmp_pointer: Same as err_pointer except it indicates the bytes of tmp file processed.

new_bytes: This is the amount of new text we need to display at every iteration.
This will be addition of the tmp_length and err_length we have recv except the tmp_pointer.
If err length is non zero, then tmp_pointer needs to be reset to the current temp_length as the err_length var will contain 
"""